Log.cpp

```diff
@@ -23,6 +23,7 @@ Log::~Log()
 {
 	logMessageStream << std::endl;
 	const auto logMessage = logMessageStream.str();
+	WriteToConsole(logLevel, logMessage); // write to the console for the sake of tests and Frontend
 	WriteToFile(logLevel, logMessage);
```

Comment:
Maybe get the time here once instead of in both `WriteToConsole` and `WriteToFile`? Might be slightly more efficient.

Then do this:
```suggestion
	WriteToConsole(timeInfo, logLevel, logMessage); // write to the console for the sake of tests and Frontend
	WriteToFile(timeInfo, logLevel, logMessage);
```

Reply:
That function writes to stream and I don't want to mess with to be honest...